src/app/containers/PageHandlers/withData.jsx

```diff
@@ -1,31 +1,46 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { shape, element } from 'prop-types';
 import articlePropTypes from '../../models/propTypes/article';
 import ErrorMain from '../ErrorMain';
 import nodeLogger from '../../helpers/logger.node';
 import deepGet from '../../helpers/json/deepGet';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import { getArticleHomeService } from '../PageViewAnalytics/labelHelpers/article';
 
 const logger = nodeLogger(__filename);
 
-// checks for data and status keys, setting default status if not found
+// checks for data, status & home keys, setting default status if not found
 const constructRenderObject = data => ({
   status: deepGet(['status'], data) || 500,
   pageData: deepGet(['pageData'], data),
+  home: deepGet(['pageData', 'metadata', 'passport', 'home'], data),
```

Comment:
You should be able to remove this from here, since `home` isn't necessary for the renderObject.

```suggestion
```